optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/anno/entity/TestdataAnnoEntitySubBrokenMissingEntityAnnotation.java

```diff
@@ -0,0 +1,38 @@
+package org.optaplanner.core.impl.testdata.domain.anno.entity;
+
+import org.optaplanner.core.api.domain.solution.PlanningEntityProperty;
+import org.optaplanner.core.api.domain.solution.PlanningScore;
+import org.optaplanner.core.api.domain.solution.PlanningSolution;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.api.score.buildin.simple.SimpleScore;
+import org.optaplanner.core.impl.domain.entity.descriptor.EntityDescriptor;
+import org.optaplanner.core.impl.domain.solution.descriptor.SolutionDescriptor;
+
+//@PlanningEntity // missing
```

Comment:
If this is intentially not a PlanningEntity, can we change that line to `// Intentionally no @PlanningEntity annotation` instead?

Diagonally scanning this looks like something that was temporarly uncommented and forgetten to be commented it.

Reply:
Yes, exactly. The comment suggested by you is much better, +1.